test/Dockerfile.testrunner.linux

```diff
@@ -1,15 +1,12 @@
 # Use this Dockerfile to create a Linux testrunner image
 #     docker build -t testrunner -f ./test/Dockerfile.testrunner.linux .
-#     docker run --rm -v /var/run/docker.sock:/var/run/docker.sock testrunner powershell -File test/test.ps1 <test.ps1 args>
+#     docker run --rm -v /var/run/docker.sock:/var/run/docker.sock testrunner test/test.ps1 <test.ps1 args>
 
-FROM ubuntu:16.04
+FROM microsoft/powershell:ubuntu16.04
```

Comment:
You may consider using a prebuilt image here - debian-stretch-docker-testrunner-63f2145-20184325094343.  The source for it is at https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/master/src/debian/stretch/docker-testrunner/Dockerfile.  Would save some time not having to build this entire thing every time.  This is being used in dotnet-docker-nightly - https://github.com/dotnet/dotnet-docker-nightly/pull/538

Reply:
I'll give that a shot

Reply:
Works great! Thanks for pointing that out @MichaelSimons